content/pt-br/docs/reference/glossary/persistent-volume-claim.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Reivindicação de Volume Persistente
```

Comment:
I think it is better the original title, but I saw on the PersistentVolumes's doc "Requisição de Volume Persistente"

@stormqueen1990 What do you think about this?

Reply:
"Requisição de Volume Persistente" would be great... I am going ahead with "Requisição de Volume Persistente" as title.